static/js/Misc.jsx

```diff
@@ -1575,6 +1577,84 @@ FollowButton.propTypes = {
 
 };
 
+const PictureUploader = ({callback, old_filename, caption}) => {
```

Comment:
Worth noting that if we stipulate a caption as a prop to this component, then its not really a generic picture uploader tool, but one that is specific to topics.